### PR TITLE
Update privacy policy

### PR DESCRIPTION
**Pull request by chreman**
* Included the statement for hypothesis in english and german
* updated the cookie message
* included a cookie domain in accordance with https://cookieconsent.insites.com/documentation/javascript-api/

### CODE REVIEW

**Comment by pkraker** [comment]
Thanks! This addresses all points but one: when the cookie is already set for the cookie consent, it should be removed, so that the message is displayed again. Thus we make sure to inform all of our users about the privacy policy changes. The procedure seems to be as follows:

* Remove the cookie as described e.g. [here](http://css-workshop.com/javascript-create-update-and-remove-cookies/).
* Set a new cookie with a new name, so that it isn't removed when the site is called again.

**Comment by chreman** [comment]
The old cookie is now expired, and a new one with the update notification created.

**Comment by pkraker** [comment]
Looks good, thanks! The only request that I would have is that the domain should be read from a global variable in the config. Otherwise, the message will always be shown on e.g. localhost.

File: components/head_standard.php [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<?php include '../config.php' ?>
```
Comment (pkraker) [inline components/head_standard.php]: Please remove this line as it triggers an error. It is not needed, as config.php is loaded by default by each top-level site and thus does not need to be included in components.

File: components/head_standard.php [hunk 2]
```diff
@@ -104,17 +105,24 @@ function getLabel($tag) {
...
 <?php }; ?>
-
+    function clearCookies (names) {
+      var i = 0, namesLength = names.length;
+      for (i; i < namesLength; i += 1) {
+        document.cookie = names[i] + '=; expires=Thu, 01 Jan 1970 00:00:01 GMT; path=/';
+      }
+    }
+    clearCookies(["cookieconsent_status"]);
+    var cookie_domain = <?php echo $COOKIE_DOMAIN ?>;
```
Comment (pkraker) [inline components/head_standard.php]: `$COOKIE_DOMAIN` domain needs to wrapped in inverted commas, otherwise it is intepreted as a constant, not a string.

**Comment by chreman** [comment]
I added a commit to resolve the two errors you detected, thanks for the review.

**Comment by pkraker** [comment]
Thanks!

**Pull request merged**